CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,8 @@
+# Contributor Code of Conduct
+
+Always be nice. When communicating online treat people the way you would if
+they were standing next to you. The project maintainers have the right and
+responsibility to maintain that standard of niceness, and you should contact
+them if it is not upheld.  Don't forget to be nice whenever representing the
+project to others outside the project.  If you are not nice, apologize. 
```

Comment:
Where should the person apologize? In public, or does in private count? How would you verify that? Should it be verified? By whom?